bin/ncu

```diff
@@ -35,7 +35,8 @@ program
     .option('-u, --upgrade', 'overwrite package file')
     .option('-x, --reject <matches>', 'exclude packages matching the given string, comma-delimited list, or regex')
     .option('-a, --upgradeAll', 'include even those dependencies whose latest version satisfies the declared semver dependency')
-    .option('--semverLevel <level>', 'find the highest version within "major" or "minor"');
+    .option('--semverLevel <level>', 'find the highest version within "major" or "minor"')
+    .option('--pre', 'Include prereleases in the final result');
```

Comment:
Add a corresponding option to the README under Advanced Options. I may modify the description to make it as clear as possible.